llvm/unittests/TargetParser/TargetParserTest.cpp

```diff
@@ -1130,7 +1130,8 @@ INSTANTIATE_TEST_SUITE_P(
                  AArch64::AEK_MTE,         AArch64::AEK_SSBS,
                  AArch64::AEK_FP16,        AArch64::AEK_FP16FML,
                  AArch64::AEK_SB,          AArch64::AEK_JSCVT,
-                 AArch64::AEK_FCMA,        AArch64::AEK_PERFMON}),
+                 AArch64::AEK_FCMA,        AArch64::AEK_PERFMON,
+                 AArch64::AEK_ETE,         AArch64::AEK_AM}),
```

Comment:
I notice there are no tests for `--print-enabled-extensions` for individual CPUs. Can we now remove these from `TargetParserTest`, which is quite annoying to keep updated, and instead add lit tests for these CPUs which can be autogenerated? Maybe in a follow up PR.

Reply:
OMG that would be amazing. Dealing with downstream diff there is really painful. It would be fantastic to have each one on their own line so that any downstream changes are purely `+ lines`.

If you do that, please make sure it's in its own commit, and as NFC as you can make it, so it's much easier to "audit" when merging.

Reply:
Yes, that's a very good idea! I'll work on migrating those tests in a follow up PR.